src/ontology/mondo.Makefile

```diff
@@ -226,6 +228,44 @@ reports/%-rare-diseases.tsv: $(ONT)-base.owl reports/%-rare-diseases.txt
 rare-disease-reports: reports/old-rare-diseases.tsv reports/new-rare-diseases.tsv
 	python ../scripts/filter_rare_disease_list.py reports/old-rare-diseases.tsv reports/new-rare-diseases.tsv reports/added-rare-disases.tsv reports/removed-rare-diseases.tsv
 
+
+#########################################
+######### Create Mondo Stats ############
+#########################################
+# Create overview Mondo stats as needed for presentations. This includes: count of disease classes, terms w/definitions, xrefs,
+# exact, related, narrow, and broad synonyms, and count of rare disease, cancer, infectious, and hereditary disease classes.
+
+# Get the current date including the timezone
+current_date := $(shell date)
```

Comment:
yeah, i have mixed thoughts about keeping it with the goal or top of the file, but choose to keep it here for now unless someone had strong objections. since it sounds like there aren't strong objections i'll leave it here